public/app/features/datasources/components/picker/DataSourceDropdown.tsx

```diff
@@ -212,19 +212,21 @@ const PickerContent = React.forwardRef<HTMLDivElement, PickerContentProps>((prop
 
   return (
     <div style={props.style} ref={ref} className={styles.container}>
-      <DataSourceList
-        {...props}
-        enableKeyboardNavigation
-        className={styles.dataSourceList}
-        current={current}
-        onChange={changeCallback}
-        filter={(ds) => matchDataSourceWithSearch(ds, filterTerm)}
-        onClickEmptyStateCTA={() =>
-          reportInteraction(INTERACTION_EVENT_NAME, {
-            item: INTERACTION_ITEM.CONFIG_NEW_DS_EMPTY_STATE,
-          })
-        }
-      ></DataSourceList>
+      <CustomScrollbar>
+        <DataSourceList
```

Comment:
Because this is used from the modal as well (and there it seems it is already wrapped in CustomScrollbar) would it make sense to move it inside the DataSourceList? (And remove the scrollbar that wraps this in the modal version).

Just a thought

Reply:
I wanted to use it from the component setting the layout which is the one knowing if the list would be scrollable 😄